subcommand_plugin_examples/c_subcommand_plugin_tutorial/temp_converter/builder.py

```diff
@@ -0,0 +1,13 @@
+# in a separate file "package/foo_build.py"
+import pathlib
+
+import cffi
+
+c_to_f = pathlib.Path(__file__).parent / "c_to_f.c"
+
+ffibuilder = cffi.FFI()
+ffibuilder.set_source("temp_converter._converter", c_to_f.read_text())
```

Comment:
The cffi module is traditionally prefixed with an `_` and Python code wraps it with a more friendly interface.